tests/node_test.cc

```diff
@@ -295,4 +295,54 @@ TEST(NodeTest, StrandSubscriber) {
     });
 }
 
+TEST(NodeTest, JobAliveToken) {
+    static constexpr std::size_t kThreadNum       = 4;
```

Comment:
Will this test work well on single-core machine?

Reply:
Yes, because we just launched 4 threads, not requiring parallelism.

The point here is trying to introduce concurrency, and making sure the strands and tokens work in this case. It is still true in the single core machine.